lib/navigation/preprocess_encoders.cpp

```diff
@@ -34,7 +38,38 @@ core::EncoderData EncodersPreprocessor::detectOutliers(const core::EncoderData e
   -also figure out return type/ what we update and update
   documentation as appropriate
   */
-  return {0, 0, 0, 0};
+ core::EncoderData encoder_data_copy = std::copy(encoder_data.begin(), encoder_data.end(), std::back_inserter(encoder_data));
```

Comment:
Looks like this needs to be split into 2 lines. You first have to initialize a container for the new data and then assign it:
`core::EncoderData encoder_data_copy;`
`std::copy(encoder_data.begin(), encoder_data.end(), std::back_inserter(encoder_data_copy);`